<ui version="4.0" >
 <class>EgtMinimalExifEditorControlsGui</class>
 <widget class="QWidget" name="EgtMinimalExifEditorControlsGui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelLongitude" >
     <property name="text" >
      <string>Longitude</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leLongitude" />
   </item>
   <item row="0" column="2" >
    <widget class="QFrame" name="longitudeControls" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>-1</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbtnLongitudeSave" >
        <property name="maximumSize" >
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnLongitudeCancel" >
        <property name="maximumSize" >
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="labelLatitude" >
     <property name="text" >
      <string>Latitude</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="leLatitude" />
   </item>
   <item row="1" column="2" >
    <widget class="QFrame" name="latitudeControls" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>-1</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pbtnLatitudeSave" >
        <property name="maximumSize" >
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbtnLatitudeCancel" >
        <property name="maximumSize" >
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
